pysiaf/tests/test_match_v2v3.py

```diff
@@ -0,0 +1,15 @@
+from pysiaf import iando
+from pysiaf.utils import tools
+from pysiaf.utils import tools2
+
+print('\n\n ************************************* BEGIN TEST **************************')
+instrument = 'NIRCam'
+apName_1 = 'NRCA1_FULL'
+apName_2 = 'NRCA5_FULL'
+location = '/itar/jwst/tel/share/SIAF_WG/Instruments/NIRCam/NIRCam_SIAF_2018-07-23.xml'
```

Comment:
You committed an STScI internal path to a public repository. We should absolutely avoid that for security reasons.

Please remove that and simply use
`siaf = pysiaf.Siaf('NIRCam')`